src/app/views/NoteCreate.svelte

```diff
@@ -113,11 +174,19 @@
     thunk.status.subscribe(status => {
       if (
         Object.values(status).length === thunk.request.relays.length &&
-        Object.values(status).every(s => s.status === PublishStatus.Pending)
+        Object.values(status).every((s: any) => s.status === PublishStatus.Pending)
```

Comment:
This is the wrong way to handle this, instead use `thunk: Thunk` where it's declared

Reply:
I don't understand this comment, here I am checking that all thunk request status are pending before sending the thunk status toaster.
Would you rather send it with different requirements, ie one request status being set at least?

Reply:
Sorry, I mean the type hint. The line where thunk is assigned should be the line to be fixed, the whole file thinks thunk is `any`